ext/log/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/timber/Timber.kt

```diff
@@ -0,0 +1,29 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
+package io.github.droidkaigi.confsched2019.timber
+
+import timber.log.Timber
+
+inline fun Timber.assert(throwable: Throwable) {
```

Comment:
Timber doesn't have helper method without message String. i think, it's helper method useful.

Reply:
It's useful!!